mdx/guides/unlocking-meta-transaction-swaps.mdx

```diff
@@ -0,0 +1,92 @@
+# Unlocking meta-transaction swaps With 0x API
+
+0x protocol has had native support for a trade to occur as a meta-transaction since V2, and we have now added this support to 0x API. This guide will walk you though integrating with 0x API in such a way that enables your users to benefit from a meta-transactions flow.
+
+**Note:** This tutorial assumes your users already have their allowances set to 0x protocol.
```

Comment:
Nice. Could link to https://0x.org/docs/guides/how-to-set-your-token-allowances#setting-allowances-on-etherscan